src/ex1.py

```diff
@@ -0,0 +1,20 @@
+def get_week_day (week: int) -> int:
+
+    day = ""
+
+    if week == 1:
+        day = "Monday"
+    elif week == 2:
+        day = "Tuesday"
+    elif week == 3:
+        day = "Wednesday"
+    elif week == 4:
+        day = "Thursday"
+    elif week == 5:
+        day = "Friday"
+    elif week == 6:
+        day = "Saturday"
+    elif week == 7:
+        day = "Sunday"
```

Comment:
invece che usare tutti questi if else, potresti creare un array ordinato di stringhe e accedere direttamente a questo usando come indice il valore intero che la funzione get_week_day riceve in input

Reply:
L'esercizio originale richiedeva di costruire la funzione con il costrutto switch, ma la cambio comunque
`/* Improve this program using a switch-case. */`

Reply:
Hai ragione! Colpa mia 😅  
Comunque così è molto più carino che con uno switch ahahah